src/d3d9/d3d9_swvp_emu.h

```diff
@@ -11,26 +11,41 @@ namespace dxvk {
   class D3D9VertexDecl;
   class D3D9DeviceEx;
 
+  struct D3D9CompactVertexElement {
+      uint8_t Stream : 4;
```

Comment:
the bitfield needs to use `uint16_t`, otherwise elements straddling byte boundaries get weird (and we already have a uint16 in this anyway).